packages/cli/src/generate/command.ts

```diff
@@ -82,7 +82,7 @@ export async function generate(options: GenerateOptions) {
   const packagePath = `${projectPath}/node_modules/@thirdweb-dev/generated-abis/dist`;
   if (!fs.existsSync(packagePath)) {
     throw new Error(
-      `Could not find files at ${projectPath}/node_modules/@thirdweb-dev/generated-abis`,
+      `Unable to cache ABIs. Please ensure that you're using @thirdweb-dev/sdk > 4.0.0`,
```

Comment:
wait, we're shipping v4?

Reply:
nope nope nope

Reply:
This version will change to whatever minimum version actually has the generated-abis package as a hard dep

Reply:
just put 'latest' version for now?